.github/workflows/update_apps.yml

```diff
@@ -0,0 +1,86 @@
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  update-apps:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Run app update scripts
+        run: |
+          #source pi-apps functions
+
+          #export all functions and variables
+          set -a
+          #make DIRECTORY equal to GITHUB_WORKSPACE, for subscripts and api functions
+          DIRECTORY=$GITHUB_WORKSPACE
+          source $GITHUB_WORKSPACE/api
+
+          #add special functions
+          get_release() {
+            curl --silent "https://api.github.com/repos/$1/releases/latest" | jq -r '.tag_name' | sed s/v//g
+          }
+
+          function validate_url(){
+            if command wget -q --spider "$1"; then
+              return 0
+            else
+              return 1
+            fi
+          }
+
+          #stop exporting functions
+          set +a
+          
+          #make sure all update scripts are executable
+          chmod +x $GITHUB_WORKSPACE/.github/workflows/updates/*.sh
+          cd $GITHUB_WORKSPACE
+          apps=( .github/workflows/updates/*.sh )
+          for app_directory in "${apps[@]}"; do
+            #make sure we are still in the main workspace (incase an update script left off elsewhere)
+            cd $GITHUB_WORKSPACE
+            export app_name="$(echo ${app_directory%.*} | sed 's:.*/::')"
+            echo "$app_name"
+            status "Checking $app_name for updates"
+            # move to app folder
+            cd "$GITHUB_WORKSPACE/apps/$app_name"
+            # run app update script
+            "$GITHUB_WORKSPACE/$app_directory"
+          done
+          cd
+          if test -f /tmp/updated_apps; then
+            sort -u /tmp/updated_apps > /tmp/updated_apps_sorted
+            updated_apps=$(cat /tmp/updated_apps_sorted)
+            rm -f /tmp/updated_apps /tmp/updated_apps_sorted
+            echo UPDATED_APPS=$updated_apps >> $GITHUB_ENV
+          fi
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          author: github-actions <41898282+github-actions[bot]@users.noreply.github.com>
+          commit-message: |
+            Update App Versions, run by GitHub Actions - Apps updated:
+
+            ${{ env.UPDATED_APPS }}
+          branch: "auto-app-updates"
+          title: "[AUTO] Update App Versions"
+          body: |
+            Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action:
+            Apps updated:
+
+            ${{ env.UPDATED_APPS }}
```

Comment:
would it be possible to make this a list?
for example:
```markdown
* app1
* app2
```

Reply:
yes, I actually store it as a list in a file, then print it as one line, that can be changed

Reply:
done with https://github.com/Botspot/pi-apps/pull/1330/commits/a30cf499989c797383d7da2da112c0ad2208a866